src/data/content/progetto/divulgazione-e-disseminazione.yaml

```diff
@@ -176,9 +190,12 @@ components:
           # alt: Alt lorem ipsum #C
           specular: true
           text: | #C
-            Crediamo nel valore della community e nella necessità di **creare collaborazione con le realtà sul campo, incoraggiare la partecipazione e la condivisione di buone pratiche**, partecipare ad eventi di divulgazione e networking. L’obiettivo è quello di condividere buone pratiche attraverso l’ascolto, la collaborazione e la partecipazione della comunità di designer e developer e grazie a eventi e conferenze, collaborazioni, community call, contributi esterni sul blog.
+            La community è essenziale in un ambito che cambia così velocemente. Una rete diffusa di professionisti esperti sul tema è la chiave per affrontare questa complessità, per promuovere la collaborazione su temi comuni, imparare da chi ‘opera’ sul campo, incoraggiare la partecipazione e la condivisione di buone pratiche.
+            
+            Partecipiamo a eventi e conferenze, sia in qualità di uditori che di relatori, con l’obiettivo di condividere punti di vista ed esperienze attraverso l’ascolto, la collaborazione e la partecipazione della comunità di designer e developer. 
+            
           ctas:
-          - label: Scopri le attività della community #C
+          - label: Esplora le attività della community #C
```

Comment:
@danielenole (cc @ilascar) per coerenza col resto del sito tutte le CTA dei 2° livelli se non ricordo male sono "Scopri...". Mi pare che solo le fasi dei kit e l'accesso agli archivi abbia "Esplora...".

Reply:
(btw no blocking, appena ho tempo in pomeriggio la mergiamo)